# HG changeset patch
# User Mike Miller <mtmiller@ieee.org>
# Date 1380855122 14400
#      Thu Oct 03 22:52:02 2013 -0400
# Node ID 76a8a179bf5b1d514e0c28679b2d2bbd18dc7895
# Parent  b41860a0bc2418db91a7465184b9073cfd5ec24f
Move Automake options into AM_INIT_AUTOMAKE

* configure.ac (AM_INIT_AUTOMAKE): Add required Automake options from
bootstrap.conf.  Add -Wno-override option since we override certain
Automake targets and variables.
* bootstrap.conf (AUTOMAKE): Delete Automake override variable.

diff --git a/bootstrap.conf b/bootstrap.conf
--- a/bootstrap.conf
+++ b/bootstrap.conf
@@ -148,22 +148,16 @@ source_base="libgnu"
 # Mercurial subrepository instead.
 use_git=false
 GNULIB_SRCDIR=gnulib-hg
 
 # Don't check for translations since we don't have any in Octave yet.
 # This avoids the need for sha1sum or compatible utility in bootstrap.
 SKIP_PO=true
 
-## Use --foreign since we auto-generate the AUTHORS file and the default
-## --gnu strictness level doesn't like it if the AUTHORS file is missing.
-
-AUTOMAKE="automake --foreign --warnings=no-portability"
-export AUTOMAKE
-
 bootstrap_post_import_hook ()
 {
   ## Check for files that automake --gnu would normally look for, except
   ## AUTHORS, which we autogenerate from the documentation files along
   ## with building the rest of Octave, and INSTALL, which is linked from
   ## gnulib/doc/INSTALL by the bootstrap script.
 
   for f in NEWS README COPYING CITATION; do
diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -36,17 +36,17 @@ AC_SUBST(OCTAVE_COPYRIGHT)
 dnl FIXME: We should auto-insert the Mercurial changeset ID into the
 dnl        AC_REVISION field whenever configure.ac is modified.
 dnl AC_REVISION($Revision: 1.603 $)
 AC_CONFIG_SRCDIR([libinterp/octave.cc])
 AC_CONFIG_HEADERS([config.h:config.in.h])
 AC_CONFIG_AUX_DIR([build-aux])
 AC_CONFIG_MACRO_DIR([m4])
 
-AM_INIT_AUTOMAKE([1.11 tar-ustar subdir-objects])
+AM_INIT_AUTOMAKE([1.11 foreign -Wno-portability -Wno-override tar-ustar subdir-objects])
 
 ## Add the option to enable silent rules, available since Automake 1.11
 ## and included by default starting with Automake 1.13.
 AM_SILENT_RULES
 
 OCTAVE_CANONICAL_HOST
 
 AC_DEFINE(OCTAVE_SOURCE, 1, [Define to 1 if this is Octave.])
